lib/utils.js

```diff
@@ -52,4 +54,17 @@ async function endAllSimulatorDaemons () {
   log.debug('Finishing ending all simulator daemons');
 }
 
-export { killAllSimulators, endAllSimulatorDaemons };
+async function simExists (udid) {
+  let devices = await getDevices();
+
+  devices = _.pairs(devices).map((pair) => {
```

Comment:
the exact output of `getDevices()` along with documentation is on the readme for node-simctl. Is it ok to link to that?